### ISSUE TITLE
Crash when adding a RGBW controller

### ISSUE THREAD

**Issue by bkockx** [athombv/com.fibaro#4]
**What is your Homey version (Settings → System)?**
0.9.1

**What is your App version**
1.0.2

**What did you try to achieve?**
Add a RGBW controller.

**What was the result?**
The app crashes and all Fibaro devices disappear.
When I remove the device via the settings page the app no longer crashes.
However there was a stale RGBW controller in the devices page. 
Removing the stale device does not work saying: "Device removed, but was not part of homey". (Or something like it.)

**What did you expect as result?**
1. No crash
2. A way to remove stale devices

**Any other remarks**
First time I try the 0.9.* sw. It seems much more responsive with respect to the Z-Wave devices.
Plus my motion sensor finally works. Good stuff!!

**Comment by bkockx** [comment]
Crash log from the app:
```
➜  com.fibaro git:(master) athom project --run
1/4 - Archiving...
2/4 - Uploading to Homey @ 10.81.72.1 (USB)...
3/4 - Running `com.fibaro`, press CTRL+C to abort...
4/4 - Debugging...

-------------------------------------------------
com.fibaro running...
TypeError: Cannot read property 'on' of undefined
    at /node_modules/homey-zwavedriver/lib/ZwaveDriver.js:247:58
    at Array.forEach (native)
    at /node_modules/homey-zwavedriver/lib/ZwaveDriver.js:229:29
    at /homey-app/wireless/zwave.js:53:12
    at process.<anonymous> (/homey-app/helpers/client.js:1:987)
    at emitTwo (events.js:87:13)
    at process.emit (events.js:172:7)
    at handleMessage (internal/child_process.js:695:10)
    at Pipe.channel.onread (internal/child_process.js:440:11)

--- INFO: com.fibaro has been killed ---
```

**Comment by WeeJeWel** [comment]
Can you make a screenshot of your device's settings (with all the Z-Wave info)?

If the device disappears due to the crash, disable the app temporarily and add your device as a Basic Z-Wave device.

**Comment by bkockx** [comment]
<img width="730" alt="screen shot 2016-07-30 at 11 53 56" src="https://cloud.githubusercontent.com/assets/6672864/17269571/5b3fa616-564c-11e6-9685-b8602059436a.png">

**Comment by bkockx** [comment]
When I disable the app and install the rgbw controller.
I am able to turn it on/off.

When I now enable the app I no longer get the crash but the RGBW controller is saying `invalid node`
<img width="308" alt="screen shot 2016-07-30 at 12 01 00" src="https://cloud.githubusercontent.com/assets/6672864/17269601/528bb310-564d-11e6-8a4a-d4980288bcb0.png">

**Comment by mbrouw** [comment]
For me the app does not crash but the RGBW controller is added as a basic Z-wave device changing it into a siple on/off switch. Same settings as in the screenshot above except (offcourse) another node id.

**Comment by WeeJeWel** [comment]
Very odd. It seems like there's another device around that is different but is using the same product id...

Can you do this for me?
1. Remove the device
2. Disable the Fibaro app
3. Open Chrome Inspector, tab 'Console'
4. Add the device (it will be added as basic)
5. Paste the Console output here

**Comment by mbrouw** [comment]
[Homey_Fibaro_RGBW_COMM.txt](https://github.com/athombv/com.fibaro/files/394376/Homey_Fibaro_RGBW_COMM.txt)

**Comment by bkockx** [comment]
Its fixed by commit 

fix #14 
55083f6
